core/src/main/scala/chisel3/experimental/hierarchy/core/Instance.scala

```diff
@@ -124,6 +124,23 @@ object Instance extends SourceInfoDoc {
     }
   }
 
+  class ImportedDefinitionExtModule(
```

Comment:
should this be private / final in some way

Reply:
Yes please make this private